extensions/amp-apester-media/0.1/monetization/companion/display.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @param {!JsonObject} media
+ * @param {AmpApesterMedia} apesterElement
+ */
+export function handleCompanionDisplay(media, apesterElement) {
+  const monetizationSettings = media['campaignData'] || {};
+  const companionRawSettings = monetizationSettings['companionOptions'] || {};
+  const {settings} = companionRawSettings || {};
+  if (
+    companionRawSettings &&
+    companionRawSettings.enabled &&
+    settings &&
```

Comment:
```suggestion
```

Reply:
From what I see, I think settings could be undefined, in case he in't appear in companionRawSettings or companionRawSettings is undefined.

Reply:
https://github.com/Qmerce/amphtml/pull/3/files#r327986161